spec/importc.dd

```diff
@@ -170,6 +170,46 @@ $(H2 $(LNAME2 d-side, ImportC from D's Point of View))
     static assert(E.min == 0 && E.max == 2);
     ---
 
+    $(H3 $(LNAME2 tag-symbols, Tag Symbols))
+
+    $(P Tag symbols are the identifiers that appear after the `struct`, `union`, and `enum`
+    keywords, (C11 6.7.2.3). In C, they are placed in a different symbol table from other
+    identifiers. This means two different symbols can use the same name:)
+
+    $(CCODE
+    int S;
+    struct S { int a, b; };
+    S = 3;
+    struct S *ps;
+    )
+
+    $(P D does not have this distinction. If the only declaration of an identifier is the tag
+    symbol, that's what it sees. If there is a tag symbol and a non-tag symbol with the same
+    identifier, it sees the non-tag symbol. This is normally not a problem, due to the common
```

Comment:
"it sees the non-tag symbol" -> the D compiler recognizes the non-tag symbol.

"a problem, due" -> a problem due